src/SlmQueueBeanstalkd/Queue/BeanstalkdQueue.php

```diff
@@ -26,9 +31,13 @@ class BeanstalkdQueue extends AbstractQueue implements BeanstalkdQueueInterface
      * @param string           $name
      * @param JobPluginManager $jobPluginManager
      */
-    public function __construct(Pheanstalk $pheanstalk, $name, JobPluginManager $jobPluginManager)
+    public function __construct(Pheanstalk $pheanstalk, $name, JobPluginManager $jobPluginManager, QueueOptions $options = null)
```

Comment:
The idea was to provide backward compatibility "just in case" someone used contructor in his/her code. Reason is a bit theoretical, but I have several personal painful memories about such situation with other libraries ;) 
About Code Sniffer warning - I ignored it earlier because it was a warning and personally I do not understand why this requirement still exists nowadays when everyone has wide screen monitor :)